src/workerd/jsg/setup.h

```diff
@@ -665,6 +668,24 @@ class Isolate: public IsolateBase {
       }
     }
 
+    // Sets an env value that will be expressed on the process.env
```

Comment:
expressed -> exposed?
maybe expand on "when nodejs-compat mode is used" (to refer to the flag)